### ISSUE TITLE
Display error messages on app startup for non-dev users

### ISSUE THREAD

**Issue by osilkin98** [lbryio/lbry-desktop#2457]
## The Issue
End users should have the ability to view whatever error caused the app to crash. 

I will have one of the installed binaries fail to start and don't know what the issue is, and since the developer tools are disabled, I can't figure it out on my own. Back in the 2000s, applications would just give you an error message or error code so you can debug it. The lack of this ability removes the end user's ability to debug it, or our ability to prevent it.

### Steps to Reproduce

1.  2.  3.

### Expected Behavior

Error messages should be displayed to the user 

### Actual Behavior

Error messages are not displayed to the user.

### Suggested Solutions

`alert(error);`

**Comment by tzarebczan** [comment]
Most users won't want to debug, they just want it to work.  Adding the last few lines of the sdk log could be helpful, but not always...and that's usually the cause of a majority of failures. I'd consider this blocked by https://github.com/lbryio/lbry/issues/1118

**Comment by seanyesmunt** [comment]
Is this for when you see the crash page? Or if it crashes before then?

**Comment by osilkin98** [comment]
This would be for when you see the crash page. You could just have a little `See Details` dropdown button that reveals the log which usually gets printed to `console.error`

**Comment by seanyesmunt** [comment]
Most errors will be pretty worthless without sourcemaps. But there is probably something we can do to improve this.

**Comment by osilkin98** [comment]
The problem I have with it is that I know it's something to do with my configuration, but the fact that I can't even see the error log limits my ability to troubleshoot it.

**Comment by seanyesmunt** [comment]
I gotcha. I guess for a quick fix you could change the errorBoundary/view.jsx component to put the error on the page. Just so you can see it.

**Comment by osilkin98** [comment]
@tzarebczan It wasn't anything to do with the daemon as it was something else. The problem is that I was trying to do something unrelated to development on the app so having to fork & spin up an electron instance of the app is very inconvenient and not something anyone would want to, if all they want to know is what the error is.

**Comment by tzarebczan** [comment]
Gotcha, closing for now. Please reopen a new issue if there's a particular action we need to take.
